### DISCUSSION TITLE
Commits from months ago dissappeared

### DISCUSSION THREAD

**Discussion by RodrigoM09** [community/community#47839] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

over 100 of my commits are gone, my streak is reset from 86 days to 47 and green squares are gone too. Anyone know why?

**Comment by shree675** [comment]
@RodrigoM09 It generally happens when you delete your repositories containing your commits. This is one possible reason.

**Reply by RodrigoM09** [reply]
I didn’t do anything but start a php lesson lol

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @RodrigoM09 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
